packages/core/src/domain/patient.ts

```diff
@@ -90,3 +91,44 @@ export interface Patient extends BaseDomain, PatientCreate {}
 export function getStatesFromAddresses(patient: Patient): USState[] {
   return patient.data.address.map(getState);
 }
+
+export function patientDataFromResource(patientResource: PatientResource): PatientData | undefined {
```

Comment:
Only noticed this now: this function works w/ a type from the SDK, so in theory it should not be on the domain layer - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#de6ccc0884ec4c8c976858ec476b34be).

A way to look into this is:
- if something on the domain imports something from other places, then the domain depends on those;
- the domain should have the least external dependencies as possible;
- we can have folders for these external dependencies (its fine for them to depend on the domain).

We could store it on a folder like `./external/ihe-sdk/...` or `./external/ihe/...`